deployment_config/dev_vars.yml

```diff
@@ -1,12 +1,7 @@
-AUTH_BASE: https://uat.hsesinfo.org
```

Comment:
Because the value is the same in every env we can just define it in manifest.yml